### PR TITLE
Add Event on Buttons

### PR DESCRIPTION
**Pull request by IfzaRasool**
- Add onClick event on buttons
- Add arithmetic operations

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,31 +1,53 @@
...
+          <button type="button" className="btn" onClick={this.handleClick}>5</button>
+          <button type="button" className="btn" onClick={this.handleClick}>6</button>
+          <button type="button" className="btn primary" onClick={this.handleClick}>-</button>
+          <button type="button" className="btn" onClick={this.handleClick}>1</button>
+          <button type="button" className="btn" onClick={this.handleClick}>2</button>
+          <button type="button" className="btn" onClick={this.handleClick}>3</button>
+          <button type="button" className="btn primary" onClick={this.handleClick}>+</button>
+          <button type="button" className="btn zero" onClick={this.handleClick}>0</button>
+          <button type="button" className="btn" onClick={this.handleClick}>.</button>
+          <button type="button" className="btn primary" onClick={this.handleClick}>=</button>
```
Comment (Gambit142) [inline src/components/calculator.js]: - [x] Kindly create another component as a child of the `Calculator` component and pass the component data as props to the child component. You could put all the buttons in an array, map through the array and pass each button as a prop to the component. You can read this article to have a better understanding https://www.geeksforgeeks.org/how-to-map-data-into-components-using-reactjs/

File: src/components/calculator.js [hunk 2]
```diff
@@ -1,31 +1,53 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
```
Comment (Gambit142) [inline src/components/calculator.js]: - [x] Optional) It is a convention to name the file of components in CamelCase just the way you would name your React component. This helps other developers who have a look at your code to distinguish between components and logic files in your root directory. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @IfzaRasool 👋, 

Good job so far!

### Highlights

- Use of state to change data ✔️ 
- Good use of logic files provided ✔️

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider the optional suggestion made under the review. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by elyor-doniyorov — APPROVED** [review]
Hi @IfzaRasool, 

In my opinion, your project milestone is complete!  There is nothing else to say other than... well done :shipit: 


Good luck! 🎉 
- [OPTIONAL] Kindly remove all the boilerplates created by the CRA(texts, images, styles) from `public `and `src `folders in order to make your codebase cleaner.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**